### PR TITLE
Replacing removed flag with scheduler profile config

### PR DESCRIPTION
**Pull request by bergerhoffer**
This PR is a continuation of #36425. These docs were removed in 4.11, so this is only applicable in 4.10 and earlier.

Version(s):
4.8-4.10

Issue:
https://bugzilla.redhat.com/show_bug.cgi?id=1993272

Link to docs preview:
http://file.rdu.redhat.com/~ahoffer/2022/BZ-1993272-410/nodes/scheduling/nodes-custom-scheduler.html#nodes-custom-scheduler-deploying_nodes-custom-scheduler

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@kasturinarra Sorry for the delay with this update. I've pulled the updates from #36425 into this new PR, since the custom scheduler docs have been removed in 4.11+. I have this update marked as for 4.8-4.10, but I wasn't positive about 4.8, so let me know if that's right or not.

I made the additional updates that @ingvagabund pointed out in that PR.

@ingvagabund @kasturinarra Please review whenever you get a chance.

Preview: http://file.rdu.redhat.com/~ahoffer/2022/BZ-1993272-410/nodes/scheduling/nodes-custom-scheduler.html#nodes-custom-scheduler-deploying_nodes-custom-scheduler

File: modules/nodes-custom-scheduler-deploying.adoc [hunk 1]
```diff
@@ -83,20 +113,20 @@ spec:
       containers:
       - command:
         - /usr/local/bin/kube-scheduler
-        - --address=0.0.0.0
-        - --leader-elect=false
         - --scheduler-name=custom-scheduler <2>
```
Comment (ingvagabund) [inline modules/nodes-custom-scheduler-deploying.adoc:116]: Is missing the --config option: https://github.com/openshift/openshift-docs/pull/36425/files#diff-bb19a74dacb381dda56a1d6ac6096b995403cc8fcfdb8c4493b77e74e05fd4cdR102

**Comment by ingvagabund** [comment]
4.8 sounds good

File: modules/nodes-custom-scheduler-deploying.adoc (hunk 1 above)
Comment (bergerhoffer) [reply modules/nodes-custom-scheduler-deploying.adoc:116]: Whoops looks like I missed some updates from the rest of the PR. Pulling those in now, thanks!

Comment (bergerhoffer) [reply modules/nodes-custom-scheduler-deploying.adoc]: @ingvagabund Updated!

**Comment by ingvagabund** [comment]
LGTM

**Comment by bergerhoffer** [comment]
Cherry picking to 4.8 and 4.9 since this is only applicable to 4.8-4.10. 

/cherrypick enterprise-4.9

**Pull request merged**